web/crux-ui/locales/en/versions.json

```diff
@@ -1,7 +1,7 @@
 {
   "versionsName": "Products - {{product}} - Versions - {{name}}",
 
-  "tips": "Create a new version for your product that you can use for different purposes. Rolling versions are ideal for continuous delivery, while Incremental ones are useful for production.",
+  "tips": "Create a new version for your product. You can select to default a version other versions will inherit images and configurations from. dyrector.io is the most suitable for semantic versioning, since it's the de facto way of version control. We recommend using that.",
```

Comment:
`You can select a default version, from which other versions will inherit images and configuration` ?

Reply:
i just followed the instructions, but i like your version

Reply:
@robot9706 suggestion looks better for me.